scripts/test-fuzzed-MAC.py

```diff
@@ -21,6 +21,9 @@ def main():
     """check if incorrect MAC hash is rejected by server"""
     conversations = {}
 
+    #aead = True # GCM
+    aead = False # CBC
```

Comment:
a separate script probably would be better idea: the authentication tag is just 16 bytes instead of the 20 bytes of SHA-1 HMAC